libmamba/tests/src/download/test_downloader.cpp

```diff
@@ -17,8 +17,9 @@ namespace mamba
         TEST_CASE("file_does_not_exist")
         {
 #ifdef __linux__
```

Comment:
Does not work elsewhere?

Reply:
It looks like a leftover from a previous PR, let's test on other platforms.